### PR TITLE
feature/sentry+pino

### PR DESCRIPTION
**Pull request by Syncretik**

### CODE REVIEW

File: pages/api/sentry-example-api.js
Comment (Riglanto) [inline pages/api/sentry-example-api.js:1]: To remove

File: pages/sentry-example-page.js
Comment (Riglanto) [inline pages/sentry-example-page.js:1]: Same

File: lib/constants.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const { NEXT_PUBLIC_URL } = process.env;
+
+export const isTestnetProd = () =>
```
Comment (Riglanto) [inline lib/constants.ts:3]: 4 possible envs? :sweat_smile: 
We also do not set NEXT_PUBLIC_URL currently
Based on our Vercel setup discussion we might go with [NEXT_PUBLIC_VERCEL_ENV](https://github.com/Glo-Foundation/glo-wallet/pull/32/files#diff-98e56006aa8453a29bc2b8bc8e6f718b3f17ba66470f3a52a5dc5bd643482e70R62

**Review by Riglanto — CHANGES_REQUESTED** [review]

File: lib/constants.ts (hunk 1 above)
Comment (Syncretik) [reply lib/constants.ts:3]: was based off of these discussions with @gglucass https://glodollar.slack.com/archives/C03CD2VKP2M/p1685102744412059

Comment (gglucass) [reply lib/constants.ts:3]: yeah imo let's keep the 4 possible envs, fien to do for now.

@Syncretik the NEXT_PUBLIC_URL still needs to be changed?

Comment (Riglanto) [reply lib/constants.ts:3]: still just use already existing utils to determine if we're deployed on prod
```
export const isProd = () => process.env.NEXT_PUBLIC_VERCEL_ENV === "production";
```

Also there was added `isLocal` method which is no used at all. As I see we just want to disable pino for local env so just use this method and remove all others?

Comment (Syncretik) [reply lib/constants.ts:3]: in fairness it was hidden in a generic `utils` script :P 
I can remove

**Review by gglucass — APPROVED** [review]
lgtm; let's meeerge

File: package.json [hunk 2]
```diff
@@ -14,7 +14,9 @@
   "dependencies": {
     "@0xsequence/wagmi-connector": "^1.1.5",
     "@next/font": "13.1.6",
+    "@sentry/nextjs": "^7.53.1",
     "@prisma/client": "^4.15.0",
+    "@toruslabs/eccrypto": "^3.0.0",
```
Comment (Riglanto) [inline package.json:19]: delete me

**Review by Riglanto — APPROVED** [review]
2 small changes to do

**Pull request merged**